### ISSUE TITLE
[V3 CogManager] Paths issues

### ISSUE THREAD

**Issue by aikaterna** [Cog-Creators/Red-DiscordBot#2330]
# Command bugs
#### Command name

This affects [p]paths and [p]removepath, and loading v3 core cogs when there is a v2 install on the same machine/vps

#### What cog is this command from?

CogManagerUI

#### What were you expecting to happen?

For removepath to be able to remove a path, also for a brand new install to have only the Install Path and the Core Path listed

#### What actually happened?

2 users in support and a few of my RC2 bots show an additional path listed that were not added at instance creation or later. For example:
```
Install Path: /home/user/.local/share/Red-DiscordBot/cogs/CogManager/cogs
Core Path: /usr/lib/python3.7/site-packages/redbot/cogs

1. /home/user/Red-DiscordBot/cogs
```
While there is a v2 install located at `/home/user/Red-DiscordBot/`, this location was not added by the user and on a new instance creation, this path was still added to the paths list automatically. If the user tries to use [p]removepath with this location in no. 1, it will say the path is removed but it never is removed from the [p]paths list.

This error makes the bot try to load v2 cogs for default v3 cogs, resulting in a traceback.

A RC2 bot on Heroku:
```
Install Path: /app/cogs/CogManager/cogs
Core Path: /app/.heroku/python/lib/python3.6/site-packages/redbot/cogs

1. /app/cogs
```
An RC2 bot on Ubuntu 18.04:
```
Install Path: /root/.local/share/Red-DiscordBot/cogs/CogManager/cogs
Core Path: /usr/local/lib/python3.6/dist-packages/redbot/cogs

1. /root/cogs
```
Strangely another instance that shares that vps/Red v3 install with that previous bot, does not have an additional path listed in 1.

#### How can we reproduce this issue?

I am not quite sure what makes this occur. Might be related to us grabbing the cwd in <https://github.com/Cog-Creators/Red-DiscordBot/blob/3b50ed819212ac77a517a24c0138c801031b1a61/redbot/core/bot.py#L114> and having a v2 install on the cwd path.

**Comment by Tobotimus** [comment]
Yeah, by default (and I believe persistently) the cog manager includes `CWD/cogs`, which most often turns out to be `$HOME/cogs`. I'm in favour of entirely removing this, given that it is never documented, creates odd behaviour in the cog manager, and I'm guessing was put in place initially for debugging reasons.

**Comment by Tobotimus** [comment]
Resolved by #2345.
